packages/website/docs/faq.md

```diff
@@ -2,187 +2,88 @@
 
 ## Where is my data hosted?
 
-We host the DocSearch data on Algolia's servers, with replications around the
-globe. You can find more details about the actual [server specs here][4], and
-more complete information in our [privacy policy][5].
+We host the DocSearch data on Algolia's servers, with replications around the globe. You can find more details about the actual [server specs here][7], and more complete information in our [privacy policy][8].
 
 ## Can I use DocSearch on non-doc pages?
 
-The free DocSearch we provide will crawl documentation pages. To use it on other
-parts of your website, you'll need to create your own Algolia account and
-either:
+The free DocSearch we provide will **only** crawl open-source projects documentation pages or technical blogs. To use it on other parts of your website, you'll need to create your own Algolia account and either:
 
 - Run the [DocSearch crawler][6] on your own
-- Use one of our other [framework integrations or API clients][7]
+- Use one of our other [framework integrations or API clients][9]
 
 ## Can you index code samples?
 
 Yes, but we do not recommend it.
 
-Code samples are a great way for humans to understand how people use a specific
-method. It often requires boilerplate code though, repeated across examples,
-which adds noise to the results.
-
-What we recommend instead is to exclude the code blocks from the indexing (by
-using the `selectors_exclude` option in your config), and instead structure your
-content so the method names are present in the headers.
+Code samples are a great way for humans to understand how people use a specific method. It often requires boilerplate code though, repeated across examples, which adds noise to the results.
 
 ## Why do I have duplicate content in my results?
 
-This can happen when you have more than one URL pointing to the same content,
-for example with `./docs`, `./docs/` and `./docs/index.html`.
-
-You set the `stop_urls` to all the patterns you want to exclude. The following
-example will exclude all URLs ending with `/` or `index.html`
-
-```json
-{
-  "stop_urls": ["/$", "/index.html$"]
-}
-```
-
-## Why are custom changes from the Algolia dashboard ineffective?
-
-Changing your setting from the dashboard might be something you want to do for
-some reasons .
-
-Every successful crawl sets the DocSearch settings. These settings will be
-overridden at the next crawl. We **do not recommend to edit anything from the
-dashboard**. These changes come from the JSON configuration itself.
+This can happen when you have more than one URL pointing to the same content, for example with `./docs`, `./docs/` and `./docs/index.html`.
 
-You can use the [custom_settings parameter][8] for such purpose.
+You set the [`exclusionPatterns`][10] to all the patterns you want to exclude.
```

Comment:
The actual solution would be to set a proper canonical

Reply:
Updated here: https://github.com/algolia/docsearch/pull/1048/commits/35de5e4aad4bab0f4ef9c0e4d35b1a73dc779fc8